controllers/productController.js

```diff
@@ -0,0 +1,14 @@
+const { viewProduct } = require('../services/productService')
+const { viewReview } = require('../services/commentsService')
+
+
+const viewProductController = async (req, res) => {
+    const id = req.params.product_id 
+    const product = await viewProduct(id)
+    if(product.length ===0) return res.status(400).json({message:'PRODUCT_NOT_FOUND'})
+    const review = await viewReview(id)
+    if(review[0].id === null) return res.status(200).json({product: product[0]})
+    res.status(200).json({product: product[0], review: review})
+}
```

Comment:
```suggestion
    const productId = req.params.product_id 
    const product = await productService.viewProduct(productId)    
    if(product.length ===0) return res.status(400).json({message:'PRODUCT_NOT_FOUND'})

    const review = await commentService.viewReview(productId)
    if(review[0].id === null) return res.status(200).json({product: product[0]})

    res.status(200).json({product: product[0], review: review})
}
```
1. productId로 명시해서 변수명 지정해 주세요.
2. viewProduct와 viewReview를 어떤 service에서 불러오는지 명확히 알 수 있도록 productService.viewProduct로 호출해 주세요.